single_role_playbooks/azure_general.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Configure general settings using Azure API.
+  hosts: all
+  connection: local
+  gather_facts: false
+  vars:
+    ansible_python_interpreter: python
```

Comment:
Why is this required on the Ansible control host; does it fail when python3 is detected on our laptops?